### ISSUE TITLE
[Bug] "wpackagist-plugin/stop-spammer-registrations-plugin": "6.15" different version between wordpress download and wpackagist

### ISSUE THREAD

**Issue by christian-heiling** [outlandishideas/wpackagist#207]
6.15 is the newest version of this wordpress plugin.

the problem is when you download the wordpress plugin directly via https://wordpress.org/plugins/stop-spammer-registrations-plugin you get a different version compared with wp-packagist.

you see the diff e.g. in readme.txt

```
--- readme.txt    (Revision 11)
+++ readme.txt    (Arbeitskopie)
@@ -34,7 +34,7 @@
 == Changelog ==

 = 6.15 =
-* Committed changes again.
+* Committed changes again. There was a version conflict and old version was comitted.

 = 6.14 =
 * fixed typo in last commit
```

but also other files are affected

**maybe a reason for that**

the plugin autor commited into this tag twice

```
> svn co https://plugins.svn.wordpress.org/stop-spammer-registrations-plugin/tags/6.15 .
> svn log
------------------------------------------------------------------------
r1324897 | kpgraham | 2016-01-09 19:19:39 +0100 (Sam, 09. Jän 2016) | 1 Zeile

fixed typo in main program
------------------------------------------------------------------------
r1324823 | kpgraham | 2016-01-09 17:27:16 +0100 (Sam, 09. Jän 2016) | 1 Zeile

Correct version mis-match
------------------------------------------------------------------------
```

think that wpackagist lost this update

**Comment by khromov** [comment]
@christian-heiling Composer caches files by URL, so if you have old broken version of 6.15 you have to clear your composer cache (delete ~/.composer/cache) and then run `composer install` again to solve your issues.

**Comment by joaquimds** [comment]
@khromov kindly provided a fix for this in #220, let us know if you have any more problems
